### ISSUE TITLE
[Bug] Error creating new model

### ISSUE THREAD

**Issue by miquelangeld** [Laravel-Backpack/devtools-issues#67]
Hi Trying to create a new model (with crud and I get this error):

SQLSTATE[IMSSP]: An invalid attribute was designated on the PDO object. [Exception](https://flareapp.io/share/NPL8k9XP)

dev tools doesn't create any file, just throw the error. I'm able to generate the model (with crud, request, migration,etc.) using CLI and then migrate without any error.+

### PHP VERSION:
PHP 8.1.22 (cli) (built: Aug 14 2023 05:32:33) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.22, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.22, Copyright (c), by Zend Technologies
    with Xdebug v3.2.1, Copyright (c) 2002-2023, by Derick Rethans

### LARAVEL VERSION:
10.20.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/backupmanager: v5.0.0
backpack/basset: 1.2.0
backpack/crud: 6.1.15
backpack/devtools: 2.0.3
backpack/editable-columns: 3.0.2
backpack/generators: v4.0.2
backpack/logmanager: v5.0.0
backpack/permissionmanager: 7.0.1
backpack/pro: 2.0.11
backpack/settings: 3.1.0
backpack/theme-coreuiv2: 1.1.4
backpack/theme-tabler: 1.0.12

### Operating System and Server Setup
I'm using Laravel Sail on Ubuntu

**Comment by karandatwani92** [comment]
Hey @miquelangeld could you please share the stack trace?

**Comment by karandatwani92** [comment]
So, you created a very basic model(id, name, timestamp).
I did the same with devtool. it worked.

is there anything else I'm missing to reproduce the error?

**Comment by miquelangeld** [comment]
I think it's a problem with certain PHP versions. Today I had the same error in my project using the MSSQL driver which never happened before. I founded this thread https://github.com/laravel/framework/issues/47937 and maybe it's a problem with some PHP versions. I'm doing some tests in order to check it.

**Comment by pxpm** [comment]
Sorry @miquelangeld MSSQL is not officially supported. 😞 

We are aware of some issue with that platform https://github.com/Laravel-Backpack/devtools-issues/issues/64

Can you try that fix and let us know if it worked for you ? 

Cheers

**Comment by miquelangeld** [comment]
Hi @pxpm Maybe I haven't explained myself. The issue that I created here is not related with MSSQL my laravel project use mysql as main  database engine. 

I just posted as additional context.  I realized that  I suddenly  had the same error interacting also with  MSSQL at the same time that devtools started to fail  (In my project I get some info from several external DBs) 

And this circumstance makes me think that the problem lies elsewhere and not in the devtools. 

I have found that in some minor versions of PHP devtools gives the mentioned error. With 8.1.22 it fails, but with 8.1.17 it doesn't. (I have two projects with laravel sail with different php versions and I have been able to test it). 

Take a look here [https://github.com/laravel/framework/issues/47937](https://github.com/laravel/framework/issues/47937)  It seems that some versions of PHP have problems when using certain database drivers, maybe it also happens with sqlite (I think that's de DB used by devtools).

I have managed to get those versions of PHP and the MSSQL driver working by adding these options to the configuration.

```
'options' => [
                PDO::ATTR_ERRMODE => PDO::ERRMODE_SILENT,
            ]
```

 But in the case of devtools I have not been able to get it to work. But I'm pretty sure that it's a PHP problem

  I hope I have explained myself better
